.gitignore

```diff
@@ -7,4 +8,4 @@ BiblioPixel.egg*
 *.pyc
 .cache
 .coverage
-htmlcov
\ No newline at end of file
+htmlcov
```

Comment:
The last line didn't end with a carriage return.  If you look at the diff you see that little funny symbol that github uses to indicate its disapproval.

It's quite likely your editor - which is what btw? - just put that in automatically, as it should.

And of course, now I looked at this, of course emacs has something to do that too, so I'll turn that on after I press return.  :-D